package-examples/helloworld/README.md

```diff
@@ -0,0 +1,53 @@
+# helloworld
+
+Sample helloworld package.  Showcases what a bare bones package is which doesn't do anything beyond declaring the current directory as a `kpt` package. 
+
+## Steps
+
+This is a simple workflow on a package that requires no configuration or
+customization.
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`
+
+  $ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/helloworld
```

Comment:
Nit: Adding one more tab makes the command also fall into verbatim.

e.g. https://github.com/GoogleContainerTools/kpt/blob/80bc310b53159977701342eb939a146e02b28964/package-examples/wordpress/README.md#view-the-package-contents

Reply:
not sure I follow.... happy to do a GVC to figure out what's missing.

Reply:
It's a minor formatting thing. Pushed a commit. You can take a look.